plugins/phishtank/komand_phishtank/actions/check/action.py

```diff
@@ -1,56 +1,39 @@
-import komand
-from .schema import CheckInput, CheckOutput
+import insightconnect_plugin_runtime
 
-# Custom imports below
-import requests
+from .schema import CheckInput, CheckOutput, Component, Input, Output
 
 
-class Check(komand.Action):
+class Check(insightconnect_plugin_runtime.Action):
     def __init__(self):
         super(self.__class__, self).__init__(
             name="check",
-            description="Submit URL to Phishtank",
+            description=Component.DESCRIPTION,
             input=CheckInput(),
             output=CheckOutput(),
         )
 
     def run(self, params={}):
-        url = params.get("url")
-        if not url:
-            raise ValueError("url is required")
 
-        url = url.strip()
+        url = params.get(Input.URL, "").strip()
         if not url.startswith("http://") and not url.startswith("https://"):
-            url = "http://" + url
+            url = f"https://{url}"
 
-        try:
-            result = self.connection.check(url)
-            self.logger.debug("result: %s", result)
-        except Exception as e:
-            self.logger.exception(e)
+        response = self.connection.phishtank_api.check(url)
+        response = response.get("results")
+
+        in_database = response.get("in_database")
+        if not in_database:
             return {
-                "url": url,
-                "in_database": False,
-                "verified": False,
+                Output.URL: response.get("url"),
+                Output.IN_DATABASE: response.get("in_database"),
```

Comment:
minor but we could just use the variable `in_database` here

Reply:
oop forgot about that one